tests/acceptance/response-details-test.js

```diff
@@ -0,0 +1,53 @@
+var chai = require('chai');
+var expect = chai.expect;
+var RSVP = require('rsvp');
+var AddonTestApp = require('ember-cli-addon-tests').AddonTestApp;
+var request = require('request');
+var get = RSVP.denodeify(request);
+
+describe('response details', function() {
+  this.timeout(300000);
+
+  var app;
+
+  before(function() {
+
+    app = new AddonTestApp();
+
+    return app.create('response')
+      .then(function() {
+        return app.startServer({
+          command: 'fastboot',
+          additionalArguments: ['--serve-assets']
+        });
+      });
+  });
+
+  after(function() {
+    return app.stopServer();
+  });
+
+  it('makes headers available via a service', function() {
+    return get({
+      url: 'http://localhost:49741/echo-request-headers',
+      headers: { 'X-FastBoot-echo': 'i-should-be-echoed-back' }
+    })
+      .then(function(response) {
+        expect(response.body).to.contain('This route echoes back request the header "X-Fastboot-Echo"');
+        expect(response.body).to.contain('i-should-be-echoed-back');
```

Comment:
The body tests are to make sure the request details were accessible to ember, only need to check response headers here